modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,40 @@
-= Section 2
+= Architecture of Red{nbsp}Hat OpenShift Data Science
+
+
+Red{nbsp}Hat OpenShift Data Science (RHODS) is a collection of tools that run on top of OpenShift.
+
+image::architecture.svg[width=800px]
```

Comment:
We need to come up with a better architecture diagram

Reply:
Added a TODO. I'll try to explore our content hubs and ask in the chats to see if we can create a better diagram